src/compute-client/src/response.rs

```diff
@@ -259,33 +290,57 @@ impl RustType<ProtoSubscribeBatch> for SubscribeBatch<mz_repr::Timestamp> {
         ProtoSubscribeBatch {
             lower: Some(self.lower.into_proto()),
             upper: Some(self.upper.into_proto()),
-            updates: self
-                .updates
-                .iter()
-                .map(|(t, r, d)| ProtoUpdate {
-                    timestamp: t.into(),
-                    row: Some(r.into_proto()),
-                    diff: *d,
-                })
-                .collect(),
+            updates: Some(proto_subscribe_batch::ProtoSubscribeBatchContents {
```

Comment:
This is not backwards compatible because it won't populate the `updates` field properly from an old version of `ProtoSubscribeBatchContents`.